examples/notepad_auto_win10.py

```diff
@@ -0,0 +1,110 @@
+# GUI Application automation and testing library
+# Copyright (C) 2006-2018 Mark Mc Mahon and Contributors
+# https://github.com/pywinauto/pywinauto/graphs/contributors
+# http://pywinauto.readthedocs.io/en/latest/credits.html
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of pywinauto nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+#OS Supported : Windows 10 only
+#Goal :: Automating few Notepad operations to test things
+#For debug purpose I have put some print statements also
+#NOTE : To make the things little slow I have given 1secs of sleep time. You can remove that if you don't want the delay.
+
+
+from pywinauto import application
+import time
+
+app=application.Application()
+app.start("Notepad.exe")
+print("Notepad started")
+
+
+app['Notepad'].wait('ready')
+print("Waiting to be ready")
+
+print("Starting jobs")
+print("Opening Page Setup Dialog")
+app['Notepad'].menu_select("File->PageSetup")
```

Comment:
Some PCs may not have this menu because there is no printer in the system. I think it can be put under condition like `if some_spec.exists():`. This is also a good showcase for method `.exists()`.

Reply:
I was trying the ```.exists() function``` but somehow I am not able to get the results. I tried the following 
```
>>> app.Print["list item"].exists()
True
```

This works fine.

But I am not able to check for the items in the list.
I tried the following also
```>>> app.Print["list item"]["Microsoft Print to PDF"].exists()```

For this I am getting this error. Maybe I am doing something wrong here. Can you help me out in this.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\dbosex\AppData\Local\Programs\Python\Python36\lib\site-packages\pywinauto-0.6.4-py3.6.egg\pywinauto\application.py", line 312, in __getitem__
AttributeError: The control does not have a __getitem__ method for item access (i.e. ctrl[key]) so maybe you have requested this in error?
```

Reply:
It may be not obvious, but for `.child_window(...)` you have to add `visible_only=False` if it's invisible item. Regarding the last error Win32 backend allows 2 levels of window specification only. For `backend="uia"` the hierarchy is different and more levels are possible.

Reply:
So it should be `app.Print["Microsoft Print to PDF"].exists()` in your case. You can check that `app.Print.children()` and `app.Print.descendants()` return the same list of the same length.